gio/src/simple_proxy_resolver.rs

```diff
@@ -0,0 +1,35 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::ProxyResolver;
+use crate::SimpleProxyResolver;
+use glib::object::IsA;
+use glib::translate::*;
+
+impl SimpleProxyResolver {
+    #[doc(alias = "g_simple_proxy_resolver_new")]
+    #[allow(clippy::new_ret_no_self)]
+    pub fn new(default_proxy: Option<&str>, ignore_hosts: Option<&[&str]>) -> ProxyResolver {
```

Comment:
Are `Some([])` and `None` handled differently for the `ignore_hosts`?

Reply:
I copied what gir autogenerated and just changed the `&[&str]`...

Reply:
If we do it manually, let's no put an `Option` here. From the C code, both are handled exactly the same and all the other APIs around `ignore_hosts` are not `nullable`.